app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -405,6 +423,135 @@ public boolean setSubtitleTrack(int index, @Nullable List<MediaStream> allStream
         return false;
     }
 
+    public int getExoPlayerTrack(MediaStreamType streamType, @Nullable List<MediaStream> allStreams) {
```

Comment:
Please add nullable notations for new (non-Kotlin) functions like the linter warning asks.